src/components/ebay-progress-bar-expressive/index.marko

```diff
@@ -0,0 +1,64 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+import { useReducedMotion } from "./component";
+$ const {
+    a11yText = "Loading...",
+    class: inputClass,
+    messages,
+    size,
+    ...htmlInput
+} = input;
+
+<div
+    ...processHtmlAttributes(htmlInput)
+    class=["progress-bar-expressive", inputClass]
+>
+    <if(messages && messages.length > 0)>
+        $ const currentMessage = messages[state.currentMessageIndex]?.text;
+        $ const nextMessage = messages[state.nextMessageIndex]?.text;
```

Comment:
Can we maybe use render body here instead of text?

So have the user do
```
<ebay-progress-bar-expressive>
   <@message>My message</@message>
   <@message>My message 2</@message>
</ebay-progress-bar-expressive>
```

And then in the code we can do
```
$ const currentMessage = messages[state.currentMessageIndex]?.renderBody
 messages[state.nextMessageIndex]?.renderBody
<!-- Later in the code -->
<${currentMessage}>
```

This is done so we can be flexible in the future in case design changes or there are other requirements.

Reply:
This is something I was wondering about! `renderBody` is definitely more flexible than a `text` string, but I also worry it could be misused -- people throwing links and icons in there, etc. But we can draw that line in our design system docs and not necessarily in the code.

Reply:
Yes that is an issue, but it does provide much more flexibility, especially if design changes to want to include other things in the messages. Im not saying that will happen, but we have seen that in the past. 
That said, I agree that if we document what can go in there in our design system docs or even in our documentation here (like storybook description for `@messages`), it should be good enough.

Reply:
Updated to use renderBody in latest commit